tests/Unit/Domain/Structure/Test_OrientationMapHelpers.cpp

```diff
@@ -567,7 +585,7 @@ void test_2d_slice_with_orientation(const OrientationMap<3>& orientation_map) {
     get(get<ScalarTensor>(expected_vars)) = oriented_mapped_coords[0];
     get<0>(get<Coords<2>>(expected_vars)) = oriented_mapped_coords[0];
     get<1>(get<Coords<2>>(expected_vars)) = oriented_mapped_coords[1];
-    CHECK(oriented_vars == expected_vars);
+    CHECK_VARIABLES_APPROX(oriented_vars, expected_vars);
```

Comment:
When I adapted the existing test to also test my new function, it changed the extents of the mesh used to fill oriented_vars which are set based on the logical coords.  This led to a roundoff difference instead of exact equality.

Reply:
OK.